src/test/java/chess/service/RoomServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package chess.service;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+
+import chess.configuration.RepositoryConfiguration;
+import chess.web.dto.RoomDto;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Import(RepositoryConfiguration.class)
+class RoomServiceTest {
```

Comment:
```suggestion
@SpringBootTest
@Import(RepositoryConfiguration.class)
class RoomServiceTest {
```

컨트롤러를 테스트하는게 아니라면 제거하는게 좋아보입니다~